client/src/i18n/en/table.json

```diff
@@ -103,6 +103,7 @@
             "EXCHANGE_RATE": "Exchange Rate",
             "EXPIRATION_DATE": "Expiration Date",
             "EXPIRE_IN":"Expire In",
+            "EXTRACT_EMPLOYEE_STANDINGS": "View extract of the employee's standings by date range",
```

Comment:
This is almost perfect!  You just need to change "standings" -> "standing".  It isn't plural in English.